Sources/WebAuthn/WebAuthnManager.swift

```diff
@@ -58,7 +58,7 @@ public struct WebAuthnManager {
     /// - Returns: Registration options ready for the browser.
     public func beginRegistration(
         user: PublicKeyCredentialUserEntity,
-        timeout: Duration? = .seconds(3600),
+        timeout: Duration? = .seconds(60),
```

Comment:
Although 60 minutes may have been too much, we should probably set this to at least 5 minutes as recommended by the [spec](https://www.w3.org/TR/webauthn-3/#sctn-createCredential):
<img width="820" alt="image" src="https://github.com/swift-server/webauthn-swift/assets/225505/4e8edf74-cb2e-4eb4-b14c-268017852841">

Reply:
(Or heck, default it to `nil` since the spec seems to also support that)

Reply:
I have no preference either way, I just came across this discrepancy and wanted to match the code with the comment (and I think I've seen this default on web's documentations as well - can't remember where).

Anyways, feel free to suggest whatever you guys think makes more sense and I'll update the PR :)

Reply:
Yeah if you could update the PR and comment to match the spec that would be the best way to do it, thanks! 🙌

Reply:
Done